external/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisBackedBlockRDD.scala

```diff
@@ -135,7 +139,8 @@ class KinesisSequenceRangeIterator(
     endpointUrl: String,
     regionId: String,
     range: SequenceNumberRange,
-    retryTimeoutMs: Int) extends NextIterator[Record] with Logging {
+    retryTimeoutMs: Int,
+    sparkConf: SparkConf) extends NextIterator[Record] with Logging {
```

Comment:
I prefer the latter. Create it in `KinesisInputDStream` and pass it down to `KinesisBackedBlockRDD`